setup.py

```diff
@@ -40,7 +40,7 @@
     # For a discussion on single-sourcing the version across setup.py and the
     # project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.1',  # Required
+    version='1.2',  # Required
```

Comment:
As @pconesa mentioned, we're updating the version number of all plugins in scipion3 to make them begin in v3.0.0.

Reply:
sure, go ahead

Reply:
fixed in 2404e6f